tools/c7n_tencentcloud/tests/test_tc_tags.py

```diff
@@ -0,0 +1,299 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+import time
+
+import pytest
+from tc_common import BaseTest
+from c7n_tencentcloud.resources.cvm import CVM
+
+
+class TestCvmTagAction(BaseTest):
+
+    @pytest.mark.vcr
+    def test_cvm_mark_op_stop(self, options, cvm):
+        policy = self.load_policy(
+            {
+                "name": "cvm-mark-for-op-stop",
+                "resource": "tencentcloud.cvm",
+                "query": [{
+                    "InstanceIds": ["ins-00lycyy6"]
+                }],
+                "actions": [
+                    {
+                        "type": "mark-for-op",
+                        "op": "stop",
+                        "days": 14
+                    }
+                ]
+            },
+            config=options
+        )
+        resources = policy.run()
+        assert not resources[0]["Tags"]
+        if self.recording:
+            time.sleep(10)
+        resources = cvm.resources()
```

Comment:
alternatively, policy.resource_manager.get_source('describe').get_resource_tags(resources) which avoids the extraneous fetch of resource state and the need for the cvm fixture.